src/Creative/InLine/Linear/MediaFile.php

```diff
@@ -5,7 +5,7 @@
 class MediaFile
 {
     const DELIVERY_PROGRESSIVE = 'progressive';
-    const DELIVERY_STREAMING = 'streaming';
+    const DELIVERY_STREAMING   = 'streaming';
```

Comment:
this is bad idea to flatten values. new valuee will require to-reflatten values again.

Reply:
Not THAT bad, actually :)
Because flattening nowdays can be done by IDE. And it's slowly becoming a standard. But NP, I'll reverse this change.

Reply:
After any change author of all lines in blame will change, and it hard to understand why, when, and why add that line.

Reply:
Makes sense.